docs/it/stime.md

```diff
@@ -0,0 +1,38 @@
+
+#### Condivisione
+
+La ricerca di pattern, abbiamo detto, è molto utile perché ci consente di riconoscere cose già viste o fatte; per quelle non conosciute il PoC e/o del tempo allocato per lo studio è un ottimo approccio. Resta però che così è vincolato alla nostra mera conoscenza dell'argomento, abbiamo un solo punto di vista che al più possiamo contaminare leggendo quello degli altri su articoli o libri ma l'opinione dal vivo racchiude molto più valore.
+Si può infrangere questa barriera effettuando dei meeting di condivisione e di stima delle entità dei problemi in gruppo; il problema visto da angolazioni differenti può svelare caratteristiche di cui non avevamo tenuto conto, anche solo un singolo confronto può portare grande valore ma più è di gruppo meglio è.
+In alcuni framework agili ci sono proprio dei momenti per questo tipo di approccio, per citarne uno: "planning poker". Una gamification di questo processo che si svolge intorno ad un tavolo dove ogni uno, usando un mazzo di carte appositamente numerato (spesso usando Fibonacci), dà la sua stima esprimendo il suo punto di vista. Si svolgono più mani proprio come a poker, ascoltare il punto di vista degli altri e le loro considerazioni consentono di arrivare ad una convergenza appunto di una visione condivisa del problema e quindi ad una stima.
+
+#### MVP
+
+Il concetto di riduzione e scomposizione porta con sé un approccio naturale, cioè definire quali siano le parti fondamentali dell'applicativo. Si può introdurre un concetto semplice ma piuttosto efficiente ed efficace: dare una risposta alla domanda "quali sono le funzionalità minime per poter andare sul mercato?"; questo accorcia di molto il tempo di arrivo sul mercato perché elimina le funzioni non necessarie, semplifica la visione e lo sviluppo. Questo concetto è alla base del pensiero agile e va sotto il nome di Minimum Viable Product (MVP), abbassa il rischio rendendo più corto lo sviluppo, chiarisce meglio i goal del progetto e aiuta ancora una volta a scomporre le cose in elementi più piccoli o, se vogliamo, minimi perché il progetto abbia quel valore sul mercato che il business si aspetta.
```

Comment:
Espanderei tutta la sezione dedicandogli qualche paragrafo in più, dall'introduzione del concetto di MVP al contesto e al risultato che questo comporta. Come sopra, è una lettura che risulterebbe sufficiente e sensata solo a chi già conosce il concetto - anche a grandi linee -, ma non al target tipico di questo contenuto. Terrei comunque il significato immutato, perché comunque è corretto, semplicemente lo espanderei per aiutare chi sente il termine per la prima, o seconda, volta.